### PR TITLE
Add component ID(132) for NAST java client plugin

### PR DESCRIPTION
**Pull request by pg-yang**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

Related issue : https://github.com/apache/skywalking/issues/9487

image

* nats-8080 is springboot application , include nats-java library
* nats://127.0.0.1:4222 is nats server address

I changed `changes.md` , because the original comment from sync ui . It's not necessary，I think.

### CODE REVIEW

File: docs/en/changes/changes.md [hunk 1]
```diff
@@ -102,7 +102,7 @@
 * Fix picking calendar with a wrong time range and setting a unique value for dashboard grid key.
 * Add PostgreSQL to Database sub-menu.
 * Implement the network profiling widget.
-* Add Nats icon for Java plugin.
```
Comment (wu-sheng) [inline docs/en/changes/changes.md:105]: This should be kept, as UI is released in main repo.

File: docs/en/changes/changes.md [hunk 2]
```diff
@@ -102,7 +102,7 @@
...
+* Add component ID(132) for NAST java client plugin
```
Comment (wu-sheng) [inline docs/en/changes/changes.md:105]: Move to backend part of changelog.

**Review by wu-sheng — CHANGES_REQUESTED** [review]

File: docs/en/changes/changes.md (hunk 2 above)
Comment (pg-yang) [reply docs/en/changes/changes.md:105]: `Add component ID(131) for Java Micronaut plugin` is also in UI part . 
The previous format of  micronaut plugin was also wrong , I will fix it  together  .
Finally , add mictonaut、nats icon should be in UI part , add component id for micronaut、nats in backend part

Comment (wu-sheng) [reply docs/en/changes/changes.md]: Yes, icon is UI part. ID is backend part.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**